app/views/periodic_expenses/new.slim

```diff
@@ -1,16 +1,22 @@
 = simple_form_for @periodic_expense do |form|
-  = form.input :name
+  = form.input :start_date
 
-  = form.input :period, collection: PeriodicExpense.periods, label_method: :humanize
+  = form.input :end_date
 
-  = form.input :amount
+  = form.input :period, collection: PeriodicExpense.periods, label_method: :humanize
```

Comment:
This has that same issue of being coupled to the models, that we've talked about.